app/src/main/java/com/google/samples/apps/sunflower/compose/PlantDetailView.kt

```diff
@@ -0,0 +1,505 @@
+                    .plus(transitionModifier)
+            } else {
+                Modifier
+            }
+            val fabAccessibilityLabel = stringResource(R.string.add_plant)
+            FloatingActionButton(
+                onClick = onFabClicked,
+                shape = SunflowerFabShape,
+                modifier = fabModifier.semantics {
+                    accessibilityLabel = fabAccessibilityLabel
+                }
+            ) {
+                Icon(Icons.Filled.Add)
+            }
+        }
+    }
+}
+
+@Composable
+private fun PlantImage(
+    scrollerPosition: ScrollerPosition,
+    imageUrl: String,
+    modifier: Modifier = Modifier
+) {
+    ParallaxEffect(scrollerPosition, modifier) { parallaxModifier ->
+        CoilImageWithCrossfade(
+            data = imageUrl,
+            contentScale = ContentScale.Crop,
+            modifier = parallaxModifier.fillMaxWidth().preferredHeight(278.dp)
+        )
+    }
+}
+
+@Composable
+private fun PlantHeaderActions(
+    onBackClicked: () -> Unit,
+    onShareClicked: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier.fillMaxSize().systemBarsPadding(top = true).padding(top = 12.dp),
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        val iconModifier = Modifier.sizeIn(maxWidth = 32.dp, maxHeight = 32.dp)
+            .drawBackground(color = Color.White, shape = CircleShape)
+
+        IconButton(
+            onClick = onBackClicked,
+            modifier = Modifier.padding(start = 12.dp).plus(iconModifier)
+        ) {
+            Icon(Icons.Filled.ArrowBack)
+        }
+        val shareAccessibilityLabel = stringResource(R.string.menu_item_share_plant)
+        IconButton(
+            onClick = onShareClicked,
+            modifier = Modifier.padding(end = 12.dp).plus(iconModifier).semantics {
+                accessibilityLabel = shareAccessibilityLabel
+            }
+        ) {
+            Icon(Icons.Filled.Share)
+        }
+    }
+}
+
+@Composable
+private fun PlantInformation(
+    name: String,
+    wateringInterval: Int,
+    description: String,
+    onNamePositioned: (LayoutCoordinates) -> Unit,
+    toolbarShown: Boolean
+) {
+    Box(modifier = Modifier.padding(24.dp)) {
+        Hide(toolbarShown) { hideModifier ->
+            Text(
+                text = name,
+                style = MaterialTheme.typography.h5,
+                modifier = Modifier.padding(start = 8.dp, end = 8.dp, bottom = 16.dp)
+                    .gravity(Alignment.CenterHorizontally).onPositioned {
+                        onNamePositioned(it)
+                    }.plus(hideModifier)
+            )
+        }
+        Text(
+            text = stringResource(id = R.string.watering_needs_prefix),
+            color = MaterialTheme.colors.primaryVariant,
+            style = MaterialTheme.typography.body1.copy(
+                fontWeight = FontWeight.Bold
```

Comment:
Should be a `fontWeight` param on `Text`

Reply:
Good catch! There's there now, it wasn't in dev13